projects/ng-core-tester/src/app/app-routing.module.ts

```diff
@@ -85,9 +85,24 @@ const canRead = (record: any): Observable<ActionStatus> => {
   });
 };
 
-/**
- * Custom configuration for form editor.
- */
+// Permissions override simple canRead, canUpdate and canDelete if defined
+const permissions = (record: any): any  => {
```

Comment:
Can be an object instead of a function, but no matter.

Reply:
done